reminder.plugin.zsh

```diff
@@ -1,10 +1,14 @@
 TODO_SAVE_TASKS_FILE="$HOME/.todo.sav"
 TODO_SAVE_COLOR_FILE="$HOME/.todo_color.sav"
 
+# Using '#' because its pretty uncommon to type
+# typeset arrays by default use ':' but that means we can't have iTerm2 clickable URLs!
+TODO_SEP="${TODO_SEP:-"\#"}"
```

Comment:
Maybe the default value should still be `:`, else it will break any existing todo file... Even if I don't know if there is a lot ^^

```suggestion
TODO_SEP="${TODO_SEP:-":"}"
```

If you have a good argument against the colon, I am open to change the default to `#`.
But this would be a breaking change and so must be considered.